services/api/database/migrations/20231213_add_advanced_task_systemwide.js

```diff
@@ -0,0 +1,21 @@
+/**
```

Comment:
This filename should probably be adjusted, the timestamp is shorter than others, and to be safe & ensure that the migrations run in the correct order, it would be good to update the filename to be in line with other migrations :safety_vest: